lib/matplotlib/backends/qt_compat.py

```diff
@@ -74,32 +77,33 @@
 
 
 def _setup_pyqt5plus():
-    global QtCore, QtGui, QtWidgets, __version__, is_pyqt5, \
+    global QtCore, QtGui, QtWidgets, QtNetwork, __version__, is_pyqt5, \
```

Comment:
I don't think we need QtNetwork anymore (and can thus revert a few changes here).

Reply:
#20882